src/MCMCMonitorDataManager/updateSequences.ts

```diff
@@ -1,25 +1,39 @@
-import { GetSequencesRequest, isGetSequencesResponse } from "../../service/src/types";
+import { GetSequencesRequest, isGetSequencesResponse, MCMCSequenceUpdate } from "../../service/src/types";
 import postApiRequest from "../networking/postApiRequest";
+import getSpaSequenceUpdates from "../spaInterface/getSpaSequenceUpdates";
 import { MCMCMonitorAction, MCMCMonitorData } from "./MCMCMonitorDataTypes";
 
-
 export default async function updateSequences(data: MCMCMonitorData, dispatch: (a: MCMCMonitorAction) => void) {
     const X = data.sequences.filter(s => (s.updateRequested))
     if (X.length > 0) {
-        const req: GetSequencesRequest = {
-            type: 'getSequencesRequest',
-            sequences: X.map(s => ({
-                runId: s.runId, chainId: s.chainId, variableName: s.variableName, position: s.data.length
-            }))
+        const runId = X[0].runId
+        if (runId.startsWith('spa|')) {
```

Comment:
This does appear in multiple places, so probably a good idea to turn it into a utility function. (We could use `SpaRun` as a subtype of `MCMCRun` and use a typeguard for it, but we won't actually have the full run object for this test anyway.)